test/NuGet.Core.Tests/NuGet.Client.Test/ContentModelTests.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.ContentModel;
+using NuGet.Frameworks;
+using NuGet.RuntimeModel;
+using Xunit;
+
+namespace NuGet.Client.Test
+{
+    public class ContentModelTests
+    {
+        [Fact]
+        public void ContentModel_LibAnyMapsToDotnet()
+        {
+            // Arrange
+            var conventions = new ManagedCodeConventions(
+                new RuntimeGraph(
+                    new List<CompatibilityProfile>() { new CompatibilityProfile("net46.app") }));
+
+            var collection = new ContentItemCollection();
+            collection.Load(new string[]
+            {
+                "lib/any/a.dll",
+            });
+
+            // Act
+            var groups = collection.FindItemGroups(conventions.Patterns.RuntimeAssemblies)
+                .OrderBy(group => ((NuGetFramework)group.Properties["tfm"]).GetShortFolderName())
+                .ToList();
+
+            // Assert
+            Assert.Equal(1, groups.Count);
+            Assert.Equal(".NETPlatform,Version=v5.0", groups[0].Properties["tfm"].ToString());
```

Comment:
Compare to `FrameworkIdentifiers.CommonFrameworks.DotNet`? This seems like a "truer" test since the string `".NETPlatform,Version=v5.0"` would also allow this to pass.